README.md

```diff
@@ -35,6 +35,19 @@ ODK Aggregate can be deployed on Google's App Engine, enabling users to quickly
 
 Aggregate supports a variety of DBs, but we strongly recommend you use PostgreSQL first to ensure everything is working. If you wish to use another DB (e.g., Google App Engine, MySQL, or SQLServer databases) after that see [database configurations](docs/database-configurations.md).
 
+### PostgreSQL with Docker using Docker Compose
+
+1. Install [Docker](https://www.docker.com) and [Docker Compose](https://docs.docker.com/compose)
+
+1. To start the development server run the gradle task `postgresqlComposeUp` with the command: `./gradlew postgresqlComposeUp`
+    * Make sure port **5432** is not used by any other service in your computer. You can change this editing the `ports` section of the `dev_db/postgresql/docker-compose-yml` configuration file. Be sure to check the documentation: [Compose file version 3 reference - Ports section](https://docs.docker.com/compose/compose-file/#ports).
```

Comment:
I'd propose `db` instead of `dev_db` so can reduce the number of folders if we spin up a production db eventually. 

The file name should be `docker-compose.yml` not `docker-compose-yml`.